test/addons/async-hooks-promise/test.js

```diff
@@ -11,16 +11,18 @@ if (process.env.NODE_TEST_WITH_ASYNC_HOOKS) {
 }
 
 // Baseline to make sure the internal field isn't being set.
+// Because of bootstrap tracking this will be enabled.
```

Comment:
What does bootstrap tracking mean here? It seems suspicious that after the refactoring the promise hooks are turned on right off the bat.

Reply:
This was leftover from the async hooks work, removed.